addons/website/controllers/main.py

```diff
@@ -276,6 +276,13 @@ def get_dynamic_filter(self, filter_id, template_key, limit=None, search_domain=
         )
         return dynamic_filter and dynamic_filter.render(template_key, limit, search_domain) or ''
 
+    @http.route('/website/snippet/options_filters', type='json', auth='public', website=True)
```

Comment:
auth='user'

Public user should never call this route, it is something linked to editor only